scripts/dns-zone-and-server-certificates/create-or-update-one-aws-dns-zone.sh

```diff
@@ -0,0 +1,112 @@
+#! /bin/bash
+
+if ( [ $# -ne 4 ] ) then
+  echo "This script create DNS zone for a PN environment and create the delegation in parent zone pn.pagopa.it"
+  echo "Usage: $0 <environment-name> <zone-profile> <parent-zone-profile> <region>"
+  echo ""
+  echo "This script require following executable configured in the PATH variable:"
+  echo " - aws cli 2.0 "
+  echo " - jq"
+
+  if ( [ "$BASH_SOURCE" = "" ] ) then
+    return 1
+  else
+    exit 1
+  fi
+fi
+
+scriptDir=$( dirname "$0" )
+
+envName=$1
+zoneStackName="${envName}-dnszone"
+parentZoneStackName="${envName}-dnszone-delegation"
+
+zoneProfile=$2
+parentZoneProfile=$3
+
+zoneRegion=$4
+parentZoneRegion=$4
+
+
+function createOrUpdateStack() {
+  profile=$1
+  region=$2
+  stackName=$3
+  templateFile=$4
+
+  shift
```

Comment:
perché?

Reply:
i parametri nel cloudformation occupano più parametri in bash, 
lo shift mi serve per pulire l'array e poi usare $@